api-client/src/utils/textTruncator.ts

```diff
@@ -0,0 +1,10 @@
+const textTruncator = (text: string, numOfCharsAllowed: number, replaceLineBreaks = true): string => {
+  const ellipsis = '...'
+  const cleanText = replaceLineBreaks ? text.replace(/\n/g, ' ') : text
+  if (cleanText.length.trim() <= numOfCharsAllowed) {
```

Comment:
```suggestion
  if (cleanText.length <= numOfCharsAllowed) {
```
Probably added by mistake

Reply:
Ah sorry my bad. Should be 
```suggestion
  if (cleanText.trim().length <= numOfCharsAllowed) {
```

Reply:
Hah, oops, your previous suggestion looked so right I didn't stop to think about it 😆

Reply:
Yeah I know, it was totally my bad haha. Luckily we have @f1sh1918 who tested again :D